docs/usage/config-validation.md

```diff
@@ -5,18 +5,42 @@ description: How to validate Renovate's configuration.
 
 # Config Validation
 
-All [`renovate` distributions](getting-started/running.md#available-distributions) contain a standalone validator program (`renovate-config-validator`) that can be used to validate Renovate's configuration.
+You can check your Renovate configuration with a standalone program called `renovate-config-validator`.
+All [`renovate` distributions](getting-started/running.md#available-distributions) include this program.
 
-The validator program checks files passed as CLI arguments.
-If no argument is given, all [default locations](configuration-options.md) (if files exist) and the `RENOVATE_CONFIG_FILE` environment variable are checked.
+## Default behavior
+
+When you run `renovate-config-validator` with no arguments it will check:
+
+- all [default locations](configuration-options.md) (if files exist)
+- the `RENOVATE_CONFIG_FILE` environment variable
+
+For example:
 
 ```console
 $ npm install --global renovate
 added 750 packages, and audited 751 packages in 51s
 $ renovate-config-validator
-INFO: Validating renovate.json
-INFO: Config validated successfully
+ INFO: Validating renovate.json
+ INFO: Config validated successfully
 ```
 
-You can configure a [pre-commit](https://pre-commit.com) hook to validate your configuration automatically.
-Please check out the [`renovatebot/pre-commit-hooks` repository](https://github.com/renovatebot/pre-commit-hooks) for more information.
+### Pass files to check as CLI arguments
+
+Or you can pass the files to check to the `renovate-config-validator` program as CLI arguments.
+You can use this to check config files with non-default names, like when you're using preset repositories.
+For example:
+
+```console
+$ npm install --global renovate
+added 750 packages, and audited 751 packages in 51s
+$ renovate-config-validator first_config.json second_config.json
+ INFO: Validating first_config_.json
+ INFO: Validating second_config.json
+ INFO: Config validated successfully
+```
```

Comment:
When I validate multiple files with the program, it's not clear which file is good and which is bad. 

I was expecting to see output like this:

```console
$ renovate-config-validator good_config.json bad_config.json
 INFO: Validating good_config_.json
 INFO: good_config.json validated successfully
 INFO: Validating bad_config.json
 INFO: bad_config.json has errors
 INFO: finished the validation run
```

I think we should:

- Always print the filename next to the validation result
- As the final step, print a "finished the validation run" message so you know the program exited cleanly

Do you want an issue from me for this?

Reply:
@rarkins can you decide if you want an issue (or new discussion) for my idea to list good and bad configs, so it's clear which file passes/fails validation? Or is the expected use-case that users only check one file at a time?

Reply:
I don't think it's intended to validate more than one at a time

Reply:
@rarkins Choose your own adventure:

1. @HonkingGoose creates issue to make config validation work with multiple files (see example output above), and updates the docs for now to say "check one file at a time".
2. @HonkingGoose only updates the docs to say "check one file at a time".

Reply:
We do use this to validate multiple files, in a repository that hosts config presets. I think it's a valuable use case.

Reply:
I tried validating multiple files with the program, and it _seems_ to work, but the output is very confusing, quoting myself:

> When I validate multiple files with the program, it's not clear which file is good and which is bad. 
> 
> I was expecting to see output like this:
> 
> ```console
> $ renovate-config-validator good_config.json bad_config.json
>  INFO: Validating good_config_.json
>  INFO: good_config.json validated successfully
>  INFO: Validating bad_config.json
>  INFO: bad_config.json has errors
>  INFO: finished the validation run
> ```
> 
> I think we should:
> 
> - Always print the filename next to the validation result
> - As the final step, print a "finished the validation run" message so you know the program exited cleanly

But the maintainer wanted me to edit the docs. And I did. 🙃

Reply:
So what's the output today?

Reply:
## Checking two files by name

```bash
$ npm install --global renovate
$ $ renovate-config-validator good-renovate.json bad-renovate.json 
 INFO: Validating good-renovate.json
 INFO: Validating bad-renovate.json
ERROR: Found errors in configuration
       "file": "bad-renovate.json",
       "errors": [
         {
           "topic": "Configuration Error",
           "message": "Invalid configuration option: packageRules[0].descriptio"
         }
       ]
```

### `good-renovate.json`

```json
{
  "extends": ["config:best-practices"],
  "packageRules": [
    {
      "description": "Wait 3 days before creating a npm update PR",
      "matchDatasources": ["npm"],
      "minimumReleaseAge": "3 days"
    }
  ]
}
```

### `bad-renovate.json`

```json
{
    "extends": ["config:best-practices"],
    "packageRules": [
      {
        "descriptio": "Wait 3 days before creating a npm update PR",
        "matchDatasources": ["npm"],
        "minimumReleaseAge": "3 days"
      }
    ]
  }
```

## Default run with good `renovate.json` in repo root

```bash
$ renovate-config-validator
 INFO: Validating renovate.json
 INFO: Config validated successfully
```

## Default run with bad `renovate.json` in repo root

```bash
$ renovate-config-validator
 INFO: Validating renovate.json
ERROR: Found errors in configuration
       "file": "renovate.json",
       "errors": [
         {
           "topic": "Configuration Error",
           "message": "Invalid configuration option: packageRules[0].descriptio"
         }
       ]
```

Reply:
@rarkins can you take a look at the comment above, and tell me if you want an issue/discussion for the Renovate config validator output?

Reply:
@HonkingGoose is the above examples a mockup of possible future behavior, or showing current?

You wrote earlier:
> it's not clear which file is good and which is bad

But to me your recent examples seem pretty clear

Reply:
That's current behavior.

Now I'm digging deeper into the error message itself, it does say: `"file": "bad-renovate.json",` so it points to the bad file. Let's keep things as they are then. 😉 Sorry for the confusion!